src/test/java/com/google/cloud/solutions/autotokenize/testing/RandomGenericRecordGenerator.java

```diff
@@ -26,12 +26,12 @@
 
   public static final String SCHEMA_STRING =
       "{"
-          + "\"type\":\"record\", "
+          + "\"type\":\"record\","
           + "\"name\":\"testrecord\","
           + "\"fields\":["
-          + "    {\"name\":\"name\",\"type\":\"string\"},"
-          + "    {\"name\":\"id\",\"type\":\"string\"}"
-          + "  ]"
+          + "{\"name\":\"name\",\"type\":\"string\"},"
+          + "{\"name\":\"id\",\"type\":\"string\"}"
+          + "]"
           + "}";
```

Comment:
The change is just formatting, may be remove the file from patch.

Reply:
Done @anantdamle